src/Blog/View/posts.twig

```diff
@@ -15,8 +22,15 @@
 		<h1>Blog</h1>
 		<ul>
 			{% for post in posts %}
-				<li>
-					<a href="/blog/{{ post.id }}">{{ post.title }}</a>
+				<li class="u-margin-medium">
+					<a class="post-box__img" href="/blog/{{ post.id }}"> <img src="{{post.url}}"/> </a>
+					<div class="post-box__description">
+					<a class="post-description__header" href="/blog/{{ post.id }}"> <h2 class="post-description__title"> {{post.title}}</h2> <span class="post-descritpion__date"> {{post.date}} </span> </a>
```

Comment:
Add "clearfix" class selector to this element because the childrens of this element are floated. It should look like this:
```
class="post-description _header clearfix"
```

Reply:
done